main.go

```diff
@@ -56,61 +56,63 @@ func (l *LocalSSH) PrepareForEval(req *proto.PrepareForEvalRequest) (*proto.Prep
-	l.logger.Debug("evaluating local ssh against policies", "policy", request.BundlePath)
+func (l *LocalSSH) Eval(policyBundle string, a runner.AddHelper) (*proto.EvalResponse, error) {
+	l.logger.Debug("evaluating local ssh against policies", "policy", policyBundle)
 	ctx := context.TODO()
 
 	startTime := time.Now()
 
-	results, err := policyManager.New(ctx, l.logger, request.BundlePath).Execute(ctx, "local_ssh", l.data)
+	results, err := policyManager.New(ctx, l.logger, policyBundle).Execute(ctx, "local_ssh", l.data)
 	if err != nil {
-		l.logger.Error("Failed to create new policyManager object", "error", err)
-		return &proto.EvalResponse{}, err
+		l.logger.Error("Failed to evaluate against policy bundle", "error", err)
+		return &proto.EvalResponse{
+			Status: proto.ExecutionStatus_FAILURE,
+		}, err
 	}
 
 	l.logger.Debug("local ssh evaluation completed", "results", results)
 
-	response := runner.NewCallableEvalResponse()
-	result := response.GetResult()
-
 	hostname := os.Getenv("HOSTNAME")
-	result.Title = fmt.Sprintf("SSH Configuration for host: %s", hostname)
 
-	for _, result := range results {
-		tasks := []*proto.Task{}
-		for _, task := range result.Tasks {
-			activities := []*proto.Activity{}
-
-			for _, activity := range task.Activities {
-				steps := []*proto.Step{}
-				for _, step := range activity.Steps {
-					steps = append(steps, &proto.Step{
-						Title:     step.Title,
-						SubjectId: "TODO",
-					})
-				}
-
-				activities = append(activities, &proto.Activity{
-					Title:       activity.Title,
-					SubjectId:   "TODO",
-					Description: activity.Description,
-					Type:        activity.Type,
-					Steps:       steps,
-					Tools:       activity.Tools,
-				})
-			}
+	assessmentResult := runner.NewCallableAssessmentResult()
+	assessmentResult.Title = fmt.Sprintf("SSH Configuration for host: %s", hostname)
 
-			tasks = append(tasks, &proto.Task{
-				Title:       task.Title,
-				Description: &task.Description,
-				//Tasks: tasks,
-				//Activities:  activities,
-			})
-		}
+	for _, result := range results {
+		// TODO: Figure out how to send back tasks again
+		// tasks := []*proto.Task{}
+		// for _, task := range result.Tasks {
+		// 	activities := []*proto.Activity{}
+
+		// 	for _, activity := range task.Activities {
+		// 		steps := []*proto.Step{}
+		// 		for _, step := range activity.Steps {
+		// 			steps = append(steps, &proto.Step{
+		// 				Title:     step.Title,
+		// 				SubjectId: "TODO",
+		// 			})
+		// 		}
+
+		// 		activities = append(activities, &proto.Activity{
+		// 			Title:       activity.Title,
+		// 			SubjectId:   "TODO",
+		// 			Description: activity.Description,
+		// 			Type:        activity.Type,
+		// 			Steps:       steps,
+		// 			Tools:       activity.Tools,
+		// 		})
+		// 	}
+
+		// 	tasks = append(tasks, &proto.Task{
+		// 		Title:       task.Title,
+		// 		Description: &task.Description,
+		// 		//Tasks: tasks,
+		// 		//Activities:  activities,
+		// 	})
+		// }
```

Comment:
Leave this comment in place for the final implementation. It'll give us a good indication of what we need when we start flexing the data types to fit better in automation.